the notebook.ipynb

```diff
@@ -3,9 +3,11 @@
   {
```

Comment:
<p>want to keep this!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YohannesDK/DAT540_Group_11/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>